experimental/generated/beats/fields.ecs.yml

```diff
@@ -1573,6 +1573,60 @@
         Note that when the file name has multiple extensions (example.tar.gz), only
         the last one should be captured ("gz", not "tar.gz").'
       example: png
+    - name: file.elf.creation_date
+      level: extended
+      type: date
+      description: Extracted when possible from the file's metadata. Indicates when
+        it was built or compiled. It can also be faked by malware creators.
+      default_field: false
+    - name: file.elf.exports
+      level: extended
+      type: object
+      description: List of exported element names and types
+      default_field: false
+    - name: file.elf.header
+      level: extended
+      type: object
+      description: Header information of the ELF file.
+      default_field: false
+    - name: file.elf.imports
+      level: extended
+      type: object
+      description: List of imported element names and types
+      default_field: false
+    - name: file.elf.number_program_headers
+      level: extended
+      type: long
+      description: Number of ELF Program Headers.
+      default_field: false
+    - name: file.elf.number_section_headers
+      level: extended
+      type: long
+      description: Number of ELF Section Headers.
+      default_field: false
+    - name: file.elf.sections
+      level: extended
+      type: object
+      description: Section information of the ELF file.
+      default_field: false
+    - name: file.elf.segment_list
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: ELF object segment list.
+      default_field: false
+    - name: file.elf.shared_libraries
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: List of shared libraries used by this ELF object
+      default_field: false
+    - name: file.elf.telfhash
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: telfhash hash for ELF files.
```

Comment:
I'd link to [relevant documentation](https://documents.trendmicro.com/assets/pdf/TB_Telfhash-%20An%20Algorithm%20That%20Finds%20Similar%20Malicious%20ELF%20Files%20Used%20in%20Linux%20IoT%20Malware.pdf).